packages/app/src/components/SQLInlineEditor.tsx

```diff
@@ -339,6 +317,7 @@ export function SQLInlineEditorControlled({
       table={table}
       value={field.value || props.defaultValue}
       connectionId={connectionId}
+      additionalSuggestions={additionalSuggestions}
```

Comment:
This adds the ability to arbitrarily add suggestions to autocomplete that aren't already based on a source. I didn't see a way to do this, but correct me if I'm wrong!